flytekit/plugins/__init__.py

```diff
@@ -15,6 +15,8 @@
 hmsclient = _lazy_loader.lazy_load_module("hmsclient")  # type: _lazy_loader._LazyLoadModule
 type(hmsclient).add_sub_module("genthrift.hive_metastore.ttypes")
 
+sagemaker_training = _lazy_loader.lazy_load_module("sagemaker-training")  # type: _lazy_loader._LazyLoadModule
+
```

Comment:
Is the hyphen right? I think you need an underwcore

Reply:
Good catch. thanks!